diff --git a/node_modules/cubejs/lib/solve.js b/node_modules/cubejs/lib/solve.js
index d263e3a..ce5e978 100644
--- a/node_modules/cubejs/lib/solve.js
+++ b/node_modules/cubejs/lib/solve.js
@@ -2,7 +2,7 @@
   var B, BL, BR, Cnk, Cube, D, DB, DBL, DF, DFR, DL, DLF, DR, DRB, F, FL, FR, Include, L, N_FLIP, N_FRtoBR, N_PARITY, N_SLICE1, N_SLICE2, N_TWIST, N_UBtoDF, N_URFtoDLF, N_URtoDF, N_URtoUL, R, U, UB, UBR, UF, UFL, UL, ULB, UR, URF, allMoves1, allMoves2, computeMoveTable, computePruningTable, faceNames, faceNums, factorial, key, max, mergeURtoDF, moveTableParams, nextMoves1, nextMoves2, permutationIndex, pruning, pruningTableParams, rotateLeft, rotateRight, value,
     indexOf = [].indexOf;
 
-  Cube = this.Cube || require('./cube');
+  Cube = (typeof this !== 'undefined' && this && this.Cube) || require('./cube');
 
   // Centers
   [U, R, F, D, L, B] = [0, 1, 2, 3, 4, 5];
